compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/types/impl/IrTypeBase.kt

```diff
@@ -59,6 +65,12 @@ data object IrStarProjectionImpl : IrStarProjection
  */
 data object IrUninitializedType : IrType() {
     override val annotations: List<IrConstructorCall> = emptyList()
+
+    override fun toString(): String {
```

Comment:
This `toString` is not necessary because `IrUninitializedType` is a data object and has the same logic autogenerated.